js_modules/dagster-ui/packages/ui-core/src/launchpad/LaunchpadAllowedRoot.tsx

```diff
@@ -173,17 +178,22 @@ export const PIPELINE_EXECUTION_ROOT_QUERY = gql`
     $pipelineName: String!
     $repositoryName: String!
     $repositoryLocationName: String!
+    $assetSelection: [AssetKeyInput!]
```

Comment:
Uh oh, do we have a naming collision issue here? I've been thinking of "asset selection" as the string used in the asset selection input, not a list of asset keys.